content/en/account_management/billing/serverless.md

```diff
@@ -35,7 +33,7 @@ This filter excludes anything that is tagged with `env:staging` or `env:test1`.
 
 ### API
 
-The AWS API is only supported for the US site's endpoint.
+{{< site-region region="us" >}}
```

Comment:
We have content here for US and UE, but not US3 or US1-FED (empty under the API heading for those two sites). Is that intentional?

Reply:
I'm not sure, but checking on it. Confirmed this API is deployed in EU.

Reply:
Edited this to link to the API docs instead